Readme.md

```diff
@@ -12,12 +12,17 @@ The *FlexMeasures Platform* is a tool for building real-time energy flexibility
 - Developing energy flexibility services (e.g. to enable demand response) is crucial, but expensive.
 - FlexMeasures reduces development costs with real-time data integrations, uncertainty models and API/UI support.
 
-![Separation of concerns ― FlexMeasures enhancing Energy Service Company services](https://seita.nl/wp-content/uploads/2021/02/SeparationOfConcerns.png)
+![Separation of concerns ― FlexMeasures enhancing Energy Service Company services](https://raw.githubusercontent.com/SeitaBV/screenshots/main/architecture/SeparationOfConcerns.png)
 
 
-The role of FlexMeasures is to enhance the services of energy service companies (ESCOs) towards energy flexibility. For instance, forecasts and schedules are made available via API (designed with [the USEF framework](https://usef.energy) in mind). Uncertainty about ongoing events is modelled accurately and for a decentralised eco-system with the [timely-beliefs framework](https://github.com/SeitaBV/timely-beliefs).
+The role of FlexMeasures is to support energy service companies (ESCOs) to offer real-time services around energy flexibility. There are three core values:
 
-![Integration view of the FlexMeasures platform architecture](https://seita.nl/wp-content/uploads/2021/02/FlexMeasures-HighLevel.png)
+1. Real-time data integration support, with advice for the rest of the day. For example, forecasts and schedules are made available via API (designed with [the USEF framework](https://usef.energy) in mind).
+2. Energy sensor and environment data have multiple sources and their forecasts are uncertain. FlexMeasures uses the [timely-beliefs library](https://github.com/SeitaBV/timely-beliefs) to model this well.
+3. Developer support ― building customer-facing services is where energy flexibility hits the road. FlexMeasures reduces developer workload with a well-documented API, plugin support, plotting support and multi-tenancy.
```

Comment:
I'd replace "plugin support" with "data visualisation", and change the order to mention extensibility last. So:
```
FlexMeasures reduces developer workload with a well-documented API, data visualisation and multi-tenancy, and supports plugins to customise and extend the platform to your needs.
```

In `SeparationOfConcerns.png` I'd replace "plots" with "charts", being a broader and more fitting category to describe the interchanged viz objects (potentially including pie charts and maps).

Reply:
Thanks. I agree, actually I have been thinking that the plugin/extensibility angle needs a much more prominent place. That is what 95% of developers reading this would actually be needing to know ― how can I build what I need and what will I have to do to get started?

So I want to be more clear here, but also our docs need to do that, namely the getting-started guide.